test/java/seedu/addressbook/commands/SortCommandTest.java

```diff
@@ -0,0 +1,95 @@
+package seedu.addressbook.commands;
+
+import static org.junit.Assert.*;
```

Comment:
Coding standard violation here. Should specify all the classes used in this file.